folsom/src/main/java/com/spotify/folsom/MemcacheClientBuilder.java

```diff
@@ -71,31 +71,34 @@
   /** Lazily instantiated singleton default executor. */
   private static final Supplier<Executor> DEFAULT_REPLY_EXECUTOR =
       Suppliers.memoize(
-          () ->
-              new ForkJoinPool(
-                  Runtime.getRuntime().availableProcessors(),
-                  ForkJoinPool.defaultForkJoinWorkerThreadFactory,
-                  new UncaughtExceptionHandler(),
-                  true));
+              () ->
+                  new ForkJoinPool(
+                      Runtime.getRuntime().availableProcessors(),
+                      ForkJoinPool.defaultForkJoinWorkerThreadFactory,
+                      new UncaughtExceptionHandler(),
+                      true))
+          ::get;
```

Comment:
Guess we could type the `Supplier` field and constants as the Guava one to avoid the `::get` calls here but I'm not sure if that's better.

Reply:
Yeah, I was considering the same. But in general I'd like to lessen our use of Guava in Folsom, and that would be a step in the wrong direction. I was also considering adding a local memoize function instead, using only java.util.function.Supplier, but decided to rather do so in a future PR.

Reply:
Makes sense. 👍